examples/patterns/navigation/subnav.html

```diff
@@ -0,0 +1,167 @@
+---
+layout: default
+title: Navigation / Sub Navigation
+category: _patterns
+---
+<header id="navigation" class="p-navigation">
+  <div class="p-navigation__banner">
+    <div class="p-navigation__logo">
+      <a class="p-navigation__link" href="#">
+        <img class="p-navigation__image" src="https://assets.ubuntu.com/v1/d96d86b5-vanilla_black-orange_hex.svg" alt="" width="95" />
+      </a>
+    </div>
+    <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+    <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+  </div>
+  <nav class="p-navigation__nav">
+    <form class="p-search-box" action="/search">
+      <input type="search" class="p-search-box__input" name="q" placeholder="Search" required="">
+      <button type="reset" class="p-search-box__reset" alt="reset"><i class="p-icon--close"></i></button>
+      <button type="submit" class="p-search-box__button" alt="search"><i class="p-icon--search"></i></button>
+    </form>
+    <span class="u-off-screen">
+      <a href="#main-content">Jump to main content</a>
+    </span>
+    <ul class="p-navigation__links" role="menu">
+      <li class="p-navigation__link p-subnav__toggle" role="menuitem" id="link-1">
+        <a href="#link-1-menu" class="p-subnav__toggle-link">
+          Link 1 <i class="p-subnav__toggle-indicator"></i>
+        </a>
+        <ul class="p-subnav u-hide" id="link-1-menu" aria-hidden="true">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+        </ul>
+      </li>
+      <li class="p-navigation__link p-subnav__toggle" role="menuitem" id="link-2">
```

Comment:
The block should be first so should be `p-subnav` here and should extend the navigation styling.

Reply:
I chose not to couple p-navigation and p-subnav. They could be used independently of each other so I avoided any tight coupling.